.github/ISSUE_TEMPLATE/feature_proposal.yaml

```diff
@@ -0,0 +1,42 @@
+name: Feature Proposal 🚀
+description: Suggest a feature to Liqo
+labels: ["feature proposal"]
+body:
+- type: textarea
+  attributes:
+    label: Is your proposed feature related to a problem?
+    description: If so, please describe the problem.
+    placeholder: A clear and concise description of what the problem is.
+  validations:
+    required: false
+- type: textarea
+  attributes:
+    label: Describe the solution you'd like
+    description: Include any specific requirements you need
+    placeholder: A clear description of what you want to happen.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Describe your proposed solution
+    description: Please complete this section if you have ideas / suggestions on how to implement the feature. We strongly recommend discussing your approach with Liqo committers before spending lots of time implementing a change.
```

Comment:
```suggestion
    description: Please complete this section if you have ideas / suggestions on how to implement the feature. We strongly recommend discussing your approach within the Liqo community before spending time implementing a change. Please consider to ask for a slot in one of the upcoming community meetings as described in [COMMUNITY-HOW-TO.md](COMMUNITY-HOW-TO.md).
```

Reply:
Thanks, I think would be better to add the COMMUNITY-HOW-TO.md file in this PR

Reply:
Like in https://github.com/liqotech/liqo/pull/2955, we should add that link when the file will be available. What I suggest is to remove that phrase and add it in https://github.com/liqotech/liqo/pull/2955 after a rebase

Reply:
Perfectly fine for me.